content/en/docs/troubleshooting.md

```diff
@@ -73,6 +73,7 @@ The output contains all the certificate information used for signing the artifac
 
 The output also contains details about the artifact that was signed, such as the digest. You can use this information to confirm that the correct artifact was signed.
 
+For more information about signatures and their components, see [the Notary signature specification](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md).
```

Comment:
Adding this reference is good. But the specification link may be changed later.

Reply:
Based on the latest discussion, the repository will be renamed from `notaryproject` to `specifications`. See [Rename notaryproject/notaryproject repository to notaryproject/specifications](https://github.com/notaryproject/.github/issues/38).